macOS/DuckDuckGo/Bookmarks/Services/LocalBookmarkStore.swift

```diff
@@ -214,20 +214,26 @@ final class LocalBookmarkStore: BookmarkStore {
 
                 var params = processedErrors.errorPixelParameters
                 params[PixelKit.Parameters.errorSource] = source
-                PixelKit.fire(DebugEvent(GeneralPixel.bookmarksSaveFailed, error: error),
-                           withAdditionalParameters: params)
+                PixelKit.fire(
+                    DebugEvent(GeneralPixel.bookmarksSaveFailed, error: error),
+                    frequency: .dailyAndCount,
```

Comment:
If you want to preserve existing pixel name and only add a daily variant, use `.dailyAndStandard` frequency.

Reply:
Will look into this shortly.  Thanks!